### DISCUSSION TITLE
Is there a way to customize the behavior of dragging a window to the corner

### DISCUSSION THREAD

**Discussion by pushkatel** [rxhanson/Rectangle#538] [Ideas]
Right now if you drag a window to any corner it places itself into that quarter of the screen. Could I instead change it to fit into the bottom 1/6 of the screen? I have an ultra wide 21:9 monitor and it would be much more powerful to be able to drag to the corner and it size smaller than 1/4 of the screen

**Comment by rxhanson** [comment]
This is an item on my backlog that always tends to get pushed back. I definitely would consider a pull request if anyone wants to give it a try.
